src/app/client/src/app/modules/discussion/components/access-discussion/access-discussion.component.spec.data.ts

```diff
@@ -37,7 +37,7 @@ export const MockResponseData = {
       },
       emitData: {
         'forumIds': '[6]',
-        'userName': 'cctn1350'
+        'usrId': 1
```

Comment:
typo error

Reply:
it's correct only right, it's in number.

Reply:
userslug is string type but the userID is integer only